libs/blocks/gnav/gnav.js

```diff
@@ -427,7 +429,7 @@ class Gnav {
 
     searchInput.addEventListener('keydown', (e) => {
       if (e.code === 'Enter') {
-        window.open(this.getHelpxLink(e.target.value, getCountry()));
+        window.open(this.getHelpxLink(e.target.value, locale.prefix, locale.geo));
```

Comment:
Instead of Hardcoding it to HelpX URL, is it possible to have helpx URL as a fallback and the author could pass that serp page url via configuration metadata on gnav ? That way we can have flexibility on A/B testing if the page undergoes some changes and we need to test the new / against the old and also some applications may need to send customers to helpx serp and would like to have their own serp ?

Reply:
Sounds like a good idea 👍

Reply:
It sounds like a story. I will file a ticket for Milo team to triage.